src/effects/effectchainslot.cpp

```diff
@@ -165,102 +583,175 @@ void EffectChainSlot::slotChainChannelStatusChanged(const QString& group,
 
 void EffectChainSlot::slotChainEffectChanged(unsigned int effectSlotNumber,
                                              bool shouldEmit) {
```

Comment:
With the EffectChain/EffectChainSlot classes consolidated, let's work on removing these overcomplicated slotChain* functions.